lib/ui/pages/account-linking/account_discovery.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:pispapp/controllers/ephemeral/account-linking/account_discovery_controller.dart';
+import 'package:pispapp/controllers/ephemeral/account-linking/available_fsp_controller.dart';
+import 'package:pispapp/controllers/flow/account_linking_flow_controller.dart';
+import 'package:pispapp/models/account.dart';
+import 'package:pispapp/repositories/firebase/consent_repository.dart';
+import 'package:pispapp/ui/theme/light_theme.dart';
+import 'package:pispapp/ui/widgets/bottom_button.dart';
+import 'package:pispapp/ui/widgets/moja_button.dart';
+import 'package:pispapp/ui/widgets/shadow_box.dart';
+import 'package:pispapp/ui/widgets/title_text.dart';
+
+class AccountDiscovery extends StatelessWidget {
+  AccountDiscovery(this.fspId, this.fspName);
```

Comment:
I think it would be better to create a single model like `FspInfo` instead of passing two strings like this

Reply:
Yeah true, back when I created this, I had not created the Fsp model object yet, I'll change that